prov/efa/src/rxr/rxr_msg.c

```diff
@@ -205,7 +211,7 @@ int rxr_msg_select_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry, int
 	 * is specified by the user for protocol switch over points.
 	 */
 	if (efa_mr_is_neuron(tx_entry->desc[0]))
-		return longread_rtm;
+		return readbase_rtm;
```

Comment:
I am not sure whether runt read will have better performance than long read for neuron. Can we make these selection logic change within cuda?

Reply:
I new revision, I limited runt read to only cuda memory.